pandas/core/generic.py

```diff
@@ -50,6 +51,9 @@
 from pandas.io.formats.printing import pprint_thing
 from pandas.tseries.frequencies import to_offset
 
+# mypy confuses the `bool()`` method of NDFrame
+_bool = bool
```

Comment:
Yea this is unfortunate and something we've seen before:

https://github.com/pandas-dev/pandas/pull/26029#discussion_r274033707

The alias is the suggested approach so no change required here I think, but cc @jreback for visibility